### ISSUE TITLE
Java Client Crashing Local Server Build on Windows

### ISSUE THREAD

**Issue by AntumDeluge** [arianne/stendhal#564]
## 🐞 Describe the bug

System info:

> OS: Windows 10 64-bit
> Java version: 17.0.6
> MySQL version: 8.0.32
> Stendhal client+server build: git + d0bf2a00b3c6cfcdbb37a266635cb9e2eed29673

The Java client is crashing my local server upon login attempt:

```
ERROR [verManager] GameServerManager        (67  ) - Unhandled exception, server will shut down.
java.lang.NoSuchMethodError: 'void marauroa.common.net.message.MessageS2CLoginSendKey.<init>(marauroa.common.net.Channel, marauroa.common.crypto.RSAP
ublicKey, java.lang.String[])'
        at marauroa.server.game.messagehandler.LoginRequestKeyHandler.process(LoginRequestKeyHandler.java:79)
        at marauroa.server.game.messagehandler.MessageDispatcher.dispatchMessage(MessageDispatcher.java:116)
        at marauroa.server.game.GameServerManager.run(GameServerManager.java:271)
```

- Tried a local build of the client & the official client download with same result.
- Tried regenerating `server.ini`.
- Tried clean builds of both server & client.
- Tried using new MySQL & H2 databases with same result.
- Web client works, only affected by Java client.
- No errors are produced by the client's console output.

**Comment by AntumDeluge** [comment]
The problem appears to be in a recent Marauroa update. Reverting 2e460b053e351978a4db03b17e8a425c180fb12a fixes the problem.

**Comment by AntumDeluge** [comment]
I think the problem is that `libs/marauroa-client-only.jar` was not updated with `libs/marauroa.jar`.

**Comment by AntumDeluge** [comment]
Fixed with 34b84d04dabfcfe0afa30f0dcea23562beddae3f
